lte/gateway/python/integ_tests/common/subscriber_db_client.py

```diff
@@ -163,6 +192,15 @@ def list_subscriber_sids(self):
         sids = ['IMSI' + sid.id for sid in sids_pb]
         return sids
 
+    def add_subscriber_with_apn(self, sid, apn, qci):
+        logging.info("Adding subscriber : %s", sid)
+        self._added_sids.add(sid)
+        num_apn = len(apn)
+        sub_data = self._get_subscriberdb_data_with_apn(sid, apn, qci, num_apn)
```

Comment:
Why do we need to pass num_apn as a parameter? It can be computed inside the function.

Reply:
done